src/videojs.hlsjs.js

```diff
@@ -150,8 +150,12 @@ var HlsSourceHandler = {
 if (Hls.isSupported()) {
   var videojs = require('video.js'); // resolved UMD-wise through webpack
 
+  // we could just use `import videojs from 'video.js'`
```

Comment:
its great that we'll support es6 modules, but the comment is not necessary
it can be `// support es6 style import`